controllers/review-status/review-status.js

```diff
@@ -199,12 +212,12 @@ function sendResponse(passon) {
             reject(error);
             return;
         } else {
-            if (process.startsWith('Blind')) {
-                passon.reviewStatus = 'Blind';
-            } else if (process.startsWith('Double blind')) {
-                passon.reviewStatus = 'Double blind';
+            if (process.toLowerCase().startsWith('blind')) {
```

Comment:
Is there a well defined list of values that DOAJ uses? If so, it would be useful to have a link to their docs from here.